wordpress-mobile/AztecEditor-iOS
# Fixing Last Standing BREAKING Unit Tests!!

Ref. #492

### To test:
- Run the unit tests. Everything *everything* should be green
- Test Inserting a List, and verify the HTML looks good

cc @diegoreymendez @SergioEstevao 
Thanks!!!
